### DISCUSSION TITLE
作者大大这种动态几何背景是用的什么？似乎不是项目本身支持的

### DISCUSSION THREAD

**Discussion by duanjr** [tangly1024/NotionNext#752] [Q&A]
如图
![image](https://user-images.githubusercontent.com/71218187/217535169-07414246-df72-4e5f-8ff7-2c891581d061.png)

**Comment by wintopic** [comment]
是这个吧
![image](https://github.com/tangly1024/NotionNext/assets/97203022/a601138a-a391-43eb-95ad-a2b7c556c9dc)
